recipes/obspy/meta.yaml

```diff
@@ -0,0 +1,65 @@
+# {% set version = "1.0.2.dev0" %}
+
+package:
+  name: obspy
+  version: {{ version }}
+
+source:
+#  fn: obspy-{{ version }}.zip
+#  url: https://pypi.python.org/packages/source/o/obspy/obspy-{{ version }}.zip
+  fn: maintenance_1.0.x.zip
+  url: https://github.com/obspy/obspy/archive/maintenance_1.0.x.zip
+
```

Comment:
@msarahan I believe they will do so with the bugfix release they are [preparing](https://github.com/conda-forge/staged-recipes/pull/582/#issuecomment-218465128). We won't merge this just yet.